### PR TITLE
Connect DB and Client

### PR DESCRIPTION
**Pull request by pon-y**

### CODE REVIEW

File: database/psdb.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const client = new Client({
+  user: credentials.username,
+  host: credentials.host,
+  database: credentials.database,
+  password: credentials.password,
+  port: credentials.port
+});
+
+let currentid = 1;
```
Comment (elizabethcan) [inline database/psdb.js:12]: why are you hard coding a property id instead of passing one in from the server?

File: server/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ app.use(bodyParser.json());
 
 // Handle GET request for index.html
 app.get('/:propertyId', (req, res) => {
-  res.sendfile(path.resolve(__dirname + '/../public/dist/index.html'));
+  res.sendFile(path.resolve(__dirname + '/../public/dist/index.html'));
 });
```
Comment (elizabethcan) [inline server/index.js:23]: you can actually remove this function entirely by replacing line 17 with `app.use('/:propertyId', serve);`

**Review by elizabethcan — APPROVED** [review]

File: sdcDataGeneration/primaryDataGeneration.js [hunk 3]
```diff
@@ -40,26 +42,26 @@ for (let j = 0; j < 10000; j++) {
...
 
     let id = i + (j * 1000);
     let photoid = i % 1000 + 1;
-    let zestimationPricea = Math.floor(Math.random() * 4500000) + 500000;
-    let thirtyDayPriceChangea = faker.random.number({'min': 15000, 'max': 50000});
-    let oneYearForcasta = faker.random.number({'min': 500000, 'max': (500000 + 100000)});
-    let comparableHomePricea = faker.random.number({'min': (500000 - 100000), 'max': (500000 + 100000)});
-    let marketAppreciationPricea = faker.random.number({'min': (500000 - 200000), 'max': (500000)});
+    let zestimationPricea = (Math.floor(Math.random() * 4500000) + 500000).toLocaleString();
+    let thirtyDayPriceChangea = faker.random.number({'min': 15000, 'max': 50000}).toLocaleString();
```
Comment (PlasmaIntec) [inline sdcDataGeneration/primaryDataGeneration.js:52]: I would recommend replacing faker.random.number with Math.number wherever possible for efficiency gains (~30% faster)

File: server/model.js [hunk 4]
```diff
@@ -33,13 +51,25 @@ module.exports = {
...
       }
-      // Grab the results of the query and clean
+      
+      //Format db data to conform to front end [int => string]
+      let cleanObj = {
+        zestimationPrice: singlePropertyData.zestimationprice,
+        thirtyDayPriceChange: singlePropertyData.thirtydaypricechange,
+        oneYearForcast: singlePropertyData.oneyearforcast,
+        comparableHomePrice: singlePropertyData.comparablehomeprice,
+        marketAppreciationPrice: singlePropertyData.marketappreciationprice
```
Comment (PlasmaIntec) [inline server/model.js:61]: Consider taking advantage of object destructuring to make this code section more readable

**Pull request merged**